pkg/cache/cache.go

```diff
@@ -19,11 +18,13 @@ type ClientInterface interface {
 
 // NewCache ...
 func NewCache(ctx context.Context, cacheEngine models.CacheEngine, config config.Config) (ClientInterface, error) {
+	ttl := 4 * config.GroupSyncInterval
+
 	switch cacheEngine {
 	case models.RedisCacheEngine:
-		return NewRedisCache(ctx, config.RedisURI, 5*time.Minute)
+		return NewRedisCache(ctx, config.RedisURI, ttl)
 	case models.MemoryCacheEngine:
-		return NewMemoryCache(5*time.Minute, 10*time.Minute)
+		return NewMemoryCache(ttl, 2*ttl)
```

Comment:
Why should the ttl be double for memory cache?

Reply:
@phillebaba it's the cleanupInterval (not when it expired, but when it's removed from memory) `(defaultExpiration time.Duration, cleanupInterval time.Duration)`